.github/workflows/basic.yml

```diff
@@ -0,0 +1,68 @@
+# Based on https://github.com/actions-rs/example/blob/master/.github/workflows/quickstart.yml
+
+on: [push, pull_request]
+
+name: Basic
+
+jobs:
```

Comment:
Assume this is the thing to replace the Cargo yml above? Is it functionally equivalent?

Reply:
Yep. :)